### DISCUSSION TITLE
Prime Video no HD playback and Netflix no playback at all

### DISCUSSION THREAD

**Discussion by Sohirim** [Alex313031/Thorium-Win#118] [Q&A]
Hey when i try to watch Prime Videos with Thorium on Windows 10 it shows that i can not watch hd videos with this browser. And Netflix is not working at all. A Browser that can not play HD videos or Videos in general is useless. Do you guys have a fix or any ideas?

**Comment by gz83** [comment]
Not all videos will not play, but if the movie site uses widevine to handle the audio and video, then I'm sorry, but Thorium may not be able to play those videos, for the exact reason, see

https://github.com/Alex313031/Thorium-Win-AVX2/issues/84
